public/_locales/cs/messages.json

```diff
@@ -0,0 +1,49 @@
+{
+	"extensionDescription": {
+		"message": "Krásně vizualizované statistiky pro vaše e-mailové účty Thunderbird"
+	},
+	"popup": {
+		"message": "",
+		"account": "Účet | Účty",
+		"messagesInFolder": "{0} messages in {1} folders"
```

Comment:
Translation is missing in all values from this line on.

Reply:
@devmount Must have pushed an old commit, I've updated the translations in commit: c22d182 and the final version is in commit cd26717.